handbook/business-operations/README.md

```diff
@@ -153,7 +153,7 @@ Certain new team members, especially in go-to-market (GTM) roles, will need paid
 | 🫧 Other role    | ❌ | ❌ | ❌ | ❌ | ❌
 | Other department | ❌ | ❌ | ❌ | ❌ | ❌
 
-> _**Warning:** Do NOT buy LinkedIn Recruiter.  Use the cheapest possible LinkedIn tier.  The goal is access to profile views and data, not InMail.  Fleet does not send InMail._
+> _**Warning:** Do NOT buy LinkedIn Recruiter.  Use the cheapest possible LinkedIn tier. Fleet has a Sales Navigator account. AEs and SDRs should ask the operations team for a license. The goal is access to profile views and data, not InMail.  Fleet does not send InMail._
```

Comment:
We should light up “ask the operations team for a license” with a link to `#intake` (it’s on the same page and describes how to submit a custom request)

Reply:
Good idea. Update made.